metadata.md

```diff
@@ -257,6 +259,8 @@ The assay method that was used to perform testing. This database will be develop
 
 -   **assayDesc**: Description of assay.
 
+-   **assayReferenceLink**: Link to standard operating procedure (assay reference method)
```

Comment:
Is this a link to another assay method? So, when someone changes a previous method, they create a new assay method and link to a previous method. 

Or,

Is this just a reference to an assay method that is published elsewhere?

Reply:
The latter, a reference to the published assay method.

Reply:
Ok. This may need just a bit of clarification but let's merge and sort out later.